### PR TITLE
{tools}[foss/2020a-rc3] networkx v2.4 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

Requirement for gmxapi extension in GROMACS 2020.1

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/8011c330a4904fe0d3cb818ff0a2d861 for a full test report.

File: easybuild/easyconfigs/n/networkx/networkx-2.4-foss-2020a-rc3-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+sources = [SOURCE_TAR_GZ]
+checksums = ['f8f4ff0b6f96e4f9b16af6b84622597b5334bf9cae8cf9b2e42e7985d5c95c64']
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.02', versionsuffix),  # required for numpy, scipy, ...
+]
+
+use_pip = True
+download_dep_fail = True
```
Comment (boegel) [inline easybuild/easyconfigs/n/networkx/networkx-2.4-foss-2020a-rc3-Python-3.8.2.eb]: @akesandgren missing `sanity_pip_check = True`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3105.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/c6828131d0a818e025ddb1d3124ee9c4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2428.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/5724e4fba1debd55f8771ee362088ed3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/5aa790645cb7ff069821389dbc47fb50 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**